### ISSUE TITLE
The impact of valid on the results

### ISSUE THREAD

**Issue by zerodohero** [vqdang/hover_net#66]
Sorry to bother you again. I wonder whether train and valid can be the same?Or could valid be part of the train?I want to reproduce the result of the paper, but I haven't got a good result. 
What I am trying to do now is to copy 100 pieces from each block generated by each picture in the train to the valid.I use the Linux platform and the environment is the same

**Comment by simongraham** [comment]
Hi. Sorry for the slight delay in responding. You can get your validation set from your training set. I.e you can randomly select a proportion of your training images for validation and then extract patches accordingly.

**Comment by simongraham** [comment]
What results are you getting? You should be able to obtain results similar to what we report in the paper.

**Comment by zerodohero** [comment]
Thank you for your reply.
Paper: 0.635 0.631 0.566 0.426
my: 0.53310 0.63373 0.51674 0.27748
Sometimes (after 50 rounds, and often in the last few rounds) the classification accuracy of the last class is 0

**Comment by simongraham** [comment]
Take a look at #61 for this

**Comment by zerodohero** [comment]
Thank you for your reply
I took a closer look at the links you gave me, but I didn't get into how the training and validation sets I mentioned were split, and my environment is exactly the same as yours.It's just that the results are quite different, as I said above.

**Comment by haiqinzhong** [comment]
@zerodohero ，hi, do you solove the problem what you said about :  Sometimes (after 50 rounds, and often in the last few rounds) the classification accuracy of the last class is 0.   Now I meet same question with you. And I wonder know whether the dataset I use in valid is error(I use test as valid).

**Comment by zerodohero** [comment]
> @zerodohero ，hi, do you solove the problem what you said about : Sometimes (after 50 rounds, and often in the last few rounds) the classification accuracy of the last class is 0. Now I meet same question with you. And I wonder know whether the dataset I use in valid is error(I use test as valid).

I have solved it myself, you should separate Val from train instead of using test, or maybe there is something wrong with your environment, is it Linux system?

**Comment by haiqinzhong** [comment]
@zerodohero ，Yes, it's Linux system. And I just get the vaild set from train set. and it get the valid_dice_Miscellaneous=0 after about 70 epochs(but when I get the valid set from test set it doesn't appear.). By the way , I get valid set from train set but not delet them in train set. should I delete them? Or what should I do.

**Comment by haiqinzhong** [comment]
@zerodohero ，I have try some methods to get valid set but all have some problems. And I really don't know how to divide the valid set now. Can you tell the detail how you divide? I'd appreciate if you teach me. Please

**Comment by caprioGirl** [comment]
@zerodohero @haiqinzhong i am facing the same issue for the Miscellaneous class, how did u guys solve it, i would be really grateful if you could share?? Thankyou
